tests/e2e/features/redirection/mainPage/redirection.feature

```diff
@@ -306,4 +306,26 @@ Feature: External Redirection on the Main Page
     Then Message "Transaction submitted" should be visible
     When I click by text " Explore ecosystem "
     Then New page has "https://ecosystem.zksync.io/" address
+
+  @id1622
+  Scenario: Check redirection for the "Send to" page (Withdraw)
+    Given I go to page "/transaction/zksync/era/withdraw"
+    When I click by "text" with "Your account" value
+    When I confirm the network switching
+    Then Element with "text" " Arriving in ~24 hours " should be "visible"
+    When I click by "text" with " Arriving in ~24 hours " value
+    Then Element with "text" " Arriving in ~24 hours " should be "clickable"
+    Then New page has "https://era.zksync.io/docs/reference/troubleshooting/withdrawal-delay.html#withdrawal-delay" address
+    
+  @id1565
```

Comment:
the same as above, no info in Allure

Reply:
https://allure.matterlabs.dev/project/17/test-cases/1565?treeId=30

Reply:
Preconditions added in Allure